### DISCUSSION TITLE
WIN/AMD solution

### DISCUSSION THREAD

**Discussion by ggardlo** [open-webui/open-webui#2090] [General]
Since you did a great job creating the only known for me solution that works fully on AMD chips in Windows, I'm curious if the same solution could be applied to other models, e.g. Stable Diffusion or Whisper, which unfortunately I couldn't run correctly on AMD/Win.

**Comment by justinh-rahb** [comment]
The odds of getting this working for Windows are quite a bit slimmer, it's not as easy even for Linux as it is for Nvidia due to the tight integration they have with Docker Desktop on Win and through their container toolkit on Linux. We could certainly look at adding a `:rocm` Docker tag I just don't know if it'll be as widely compatible as CUDA.

**Comment by ggardlo** [comment]
On my 7900XTX SD and whisper works perfect on Linux as Pytorch has a compatible version with ROCm. But is only Linux version of this solution. So I thought maybe linux based Pytorch/ROCm container could do the job but I can't get direct GPU acces.
